### PR TITLE
Add Romy integration documentation

### PR DESCRIPTION
**Pull request by xeniter**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93750 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4413
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/romy.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - binary_sensor
+  - sensor
+  - vacuum
+  - button
+ha_integration_type: integration
+---
+
+The `romy` integration allows you to control your [ROMY](https://romyrobot.eu/) vacuum robot.
+
+This integration supports currently following models:
```
Comment (FletcherAU) [inline source/_integrations/romy.markdown]:
```suggestion
This integration currently supports the following models:
```

File: source/_integrations/romy.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- dustbin present
+- watertank present
+- watertank empty
+
+
+## Troubleshooting
+
+### local HTTP interface password
+
+You have to enable the local interface first with a password, this is printend as QR Code on a label directly under the dustbin inside the robot.
```
Comment (FletcherAU) [inline source/_integrations/romy.markdown]:
```suggestion
You have to enable the local interface first with a password, this is printed as QR Code on a label directly under the dustbin inside the robot.
```

File: source/_integrations/romy.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### ROMY robot interface protocol
+
+Is available as download under [romy-robot-interface-protocol](https://romyrobot.eu/pages/romy-robot-interface-protocol)
```
Comment (FletcherAU) [inline source/_integrations/romy.markdown]: This link currently redirects to the ROMY homepage. Has the download moved?

**Review by FletcherAU — CHANGES_REQUESTED** [review]

File: source/_integrations/romy.markdown (hunk 3 above)
Comment (xeniter) [reply source/_integrations/romy.markdown]: homepage url has changed, corrected it now

File: source/_integrations/romy.markdown [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+ha_release: 2023.5
+ha_config_flow: true
+ha_codeowners:
+  - '@xeniter'
+ha_domain: romy
+ha_platforms:
+  - binary_sensor
+  - sensor
+  - vacuum
+  - button
```
Comment (edenhaus) [inline source/_integrations/romy.markdown]:
```suggestion
  - vacuum
```

PR includes only the vacuum platform

File: source/_integrations/romy.markdown [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: ROMY
+description: Integrate your ROMY vacuum robot with Home Assistant.
+ha_category:
+  - Vacuum
+ha_iot_class: Local Polling
+ha_release: 2023.5
```
Comment (edenhaus) [inline source/_integrations/romy.markdown]:
```suggestion
ha_release: 2024.2
```

File: source/_integrations/romy.markdown [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Currently supported sensors are:
+
+- battery level
+- wifi rssi level
+- dustbin level
+- docked
+- dustbin present
+- watertank present
+- watertank empty
+
```
Comment (edenhaus) [inline source/_integrations/romy.markdown]:
```suggestion
```

**Review by edenhaus — DISMISSED** [review]

File: source/_integrations/romy.markdown [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+ha_domain: romy
+ha_platforms:
+  - binary_sensor
+  - sensor
+  - vacuum
+  - button
+ha_integration_type: integration
+---
+
+The `romy` integration allows you to control your [ROMY](https://www.romyrobot.com) vacuum robot.
```
Comment (c0ffeeca7) [inline source/_integrations/romy.markdown]:
```suggestion
The **ROMY** integration allows you to control your [ROMY](https://www.romyrobot.com) vacuum robot.
```
Tiny tweak. We spell the integration name the way it is shown in the UI.

File: source/_integrations/romy.markdown [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+- dustbin level
+- docked
+- dustbin present
+- watertank present
+- watertank empty
+
+
+## Troubleshooting
+
+### local HTTP interface password
```
Comment (c0ffeeca7) [inline source/_integrations/romy.markdown]:
```suggestion
### Local HTTP interface password
```

File: source/_integrations/romy.markdown [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+You have to enable the local interface first with a password, this is printed as QR Code on a label directly under the dustbin inside the robot.
```
Comment (c0ffeeca7) [inline source/_integrations/romy.markdown]:
```suggestion
You have to enable the local interface first with a password. This is printed as QR Code on a label directly under the dust bin inside the robot.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @xeniter :+1:

**Pull request merged**